### PR TITLE
bug 1683029 clarifying a heading

### PR DESCRIPTION
**Pull request by kalexand-rh**
From https://bugzilla.redhat.com/show_bug.cgi?id=1683029

@qinpingli, is this clearer?

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@openshift/team-documentation, PTAL

File: modules/cli-logging-in-kubeadmin.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@
 // * installing-aws/installing-customizations-cloud.adoc
 
 [id='cli-logging-in-kubeadmin-{context}']
-= Logging in to the `oc` CLI with the `kubeadmin` credentials
+= Logging into the cluster as the `kubeadmin` user by using the `oc` CLI
 
 You can log in to the `oc` command line interface (CLI) by using the
```
Comment (qinpingli) [inline modules/cli-logging-in-kubeadmin.adoc]: Here, need to update too.

**Comment by kalexand-rh** [comment]
@qinpingli, will you please check again?

File: modules/cli-logging-in-kubeadmin.adoc [hunk 2]
```diff
@@ -4,10 +4,10 @@
 // * installing-aws/installing-customizations-cloud.adoc
 
 [id='cli-logging-in-kubeadmin-{context}']
-= Logging in to the `oc` CLI with the `kubeadmin` credentials
+= Logging into the cluster as the `kubeadmin` user by using the `oc` CLI
 
-You can log in to the `oc` command line interface (CLI) by using the
-default `kubeadmin` user.
+You can log into cluster as the default `kubeadmin` user by using the `oc`
```
Comment (ahardin-rh) [inline modules/cli-logging-in-kubeadmin.adoc]: the cluster (or your cluster)

**Comment by ahardin-rh** [comment]
@kalexand-rh One minor comment. Looks good! :+1:

**Comment by qinpingli** [comment]
LGTM

**Pull request merged**